autoload/neomake/utils.vim

```diff
@@ -68,7 +68,9 @@ function! neomake#utils#WideMessage(msg) abort " {{{2
     "convert tabs to spaces so that the tabs count towards the window
     "width as the proper amount of characters
     let chunks = split(msg, "\t", 1)
+    " vint: -ProhibitUnnecessaryDoubleQuote
     let msg = join(map(chunks[:-2], 'v:val . repeat(" ", &tabstop - strwidth(v:val) % &tabstop)'), '') . chunks[-1]
+    " vint: +ProhibitUnnecessaryDoubleQuote
```

Comment:
There's a vint issue, where it does enable the warning also when called with `--error` in this case: https://github.com/Kuniwak/vint/issues/180.
I think you should just change it to:

    let msg = join(map(chunks[:-2], "v:val . repeat(' ', &tabstop - strwidth(v:val) % &tabstop)"), '') . chunks[-1] 

And not fiddle with controlling Vint through comments.